MekHQ/src/mekhq/campaign/finances/Loan.java

```diff
@@ -572,7 +572,12 @@ public static String randomRefNumber() {
         }
         return buffer.toString();
     }
-    
+
+    @Override
+    public int hashCode() {
+        return institution.hashCode() + 13 * refNumber.hashCode();
+    }
+
```

Comment:
This is not related to images or the other changes – it's just that I noticed this class overrides `equals()` but not `hashCode()`